### DISCUSSION TITLE
Glass Rendering iOS

### DISCUSSION THREAD

**Discussion by NiklasS97** [google/model-viewer#4822] [Q&A]
Hello everyone,

I’m kindly asking for help. Maybe someone in the community can help me. I created my AR scene in the model editor and everything works perfectly. However, I face one issue when placing the 3d Model in space with my iPhone. The glass of the model always renders as some kind of non translucent material. When in normal viewer it looks fine. Can someone help me out?

Thanks a lot!

**Comment by danilokrulj** [comment]
Glass can be a tricky one. I think if you could upload the model in question that would be very helpful. If you cant upload the model for company reasons maybe recreate a similar model showing the problem? Also if its just on iOS maybe you will have to convert the model to usdz manually to make it work as expected.

**Reply by elalish** [reply]
@AdaRoseCannon any hope of GLB support in Quick Look so we can avoid these file conversion losses and inconsistencies?
